packages/core/src/events/handlers/createEvaluationResultJob.ts

```diff
@@ -25,10 +31,32 @@ export const createEvaluationResultJob = async ({
   const providerLog = await unsafelyFindProviderLogByUuid(providerLogUuid)
   if (!providerLog) throw new NotFoundError('Provider log not found')
 
-  await createEvaluationResult({
+  const evaluationResult = await createEvaluationResult({
     evaluation,
     documentLog,
     providerLog,
     result: (response as ChainObjectResponse).object,
   }).then((r) => r.unwrap())
+  evaluationResult.id
+
+  const { evaluationResultsScope, baseQuery } = createEvaluationResultQuery(
+    evaluation.workspaceId,
+    database,
+  )
+  const result = await baseQuery
+    .where(eq(evaluationResultsScope.id, evaluationResult.id))
+    .limit(1)
+
+  const evaluationResultWithMetadata = result[0]!
+
+  websockets.emit('evaluationResultCreated', {
+    workspaceId: evaluation.workspaceId,
+    data: {
+      documentUuid: event.data.documentUuid,
+      workspaceId: evaluation.workspaceId,
+      evaluationId: evaluation.id,
+      evaluationResultId: evaluationResult.id,
+      row: evaluationResultWithMetadata,
+    },
+  })
```

Comment:
this is a side effect that should have gone into an event handler that reacts to the evaluationResultCreated aevent